src/orchestrator.js

```diff
@@ -64,7 +72,7 @@ function parseArgumentsIntoConfig(rawArgs) {
 
 function overWriteConfig(args) {
   let configFile = args["--config"] || path.resolve(__dirname, "config.json");
-  let config = JSON.parse(fs.readFileSync(configFile));
+  let config = JSON.parse(fs.readFileSync(configFile, "utf-8"));
```

Comment:
Same as above, `path.resolve`, `JSON.parse`, `fs.readFileSync` might throw an exception but I won't point out that anymore 😀

Also `let` should be replaced with `const` but also not a big deal.

Reply:
there is config file by default